derex/runner/mysql.py

```diff
@@ -168,3 +209,31 @@ def reset_mysql_openedx(project: Project, dry_run: bool = False):
         project=project,
         dry_run=dry_run,
     )
+
+
+def reset_mysql_password(current_password: str):
+    """Reset the mysql root user password."""
+    if not check_services(["mysql"]):
+        raise RuntimeError(
+            "Mysql service not found.\nMaybe you forgot to run\nddc-services up -d"
+        )
+    logger.info(f'Resetting password for mysql user "{MYSQL_ROOT_USER}"')
+
+    run_ddc_services(
+        [
+            "run",
```

Comment:
This will work only if the mysql container is already running. We can speed things up by not creating a new container, using `exec` instead of `run`